src/pip/_internal/commands/cache.py

```diff
@@ -24,13 +24,13 @@ class CacheCommand(Command):
 
     Subcommands:
 
-        dir: Show the cache directory.
-        info: Show information about the cache.
-        list: List filenames of packages stored in the cache.
-        remove: Remove one or more package from the cache.
-        purge: Remove all items from the cache.
+    - dir: Show the cache directory.
+    - info: Show information about the cache.
+    - list: List filenames of packages stored in the cache.
+    - remove: Remove one or more package from the cache.
+    - purge: Remove all items from the cache.
 
-        <pattern> can be a glob expression or a package name.
+    <pattern> can be a glob expression or a package name.
```

Comment:
Probably want to add ` `` ` around `<pattern>`.

Reply:
Good call!

```suggestion
    ``<pattern>`` can be a glob expression or a package name.
```

Reply:
> Good call!

Thanks for the quick fix.